casepropods/family_connect_subscription/plugin.py

```diff
@@ -61,8 +61,9 @@ def read_data(self, params):
                 subscription_data['rows'].append({
                     "name": "Message Set", "value": message_set["short_name"]})
                 if subscription['active'] and message_set['next_set']:
-                    actions.append(self.get_messageset_action(
-                        message_set, subscription['id']))
+                    subscription_data['actions'].append(
+                        self.get_messageset_action(
+                            message_set, subscription['id']))
```

Comment:
@rudigiesler we check if there is a `next_set` in line 63, so if it's `None` then it will just skip this block and not add the Switch button.